lib/public/views/Logs/Create/index.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+import { h } from '/js/src/index.js';
+
+/**
+ * A function to construct the create log screen
+ * @param {object} model Pass the model to access the defined functions
+ * @return {vnode} Return the view of the inputs
+ */
+const createScreen = (model) => {
+    const title = model.logs.getTitle();
+    const text = model.logs.getText();
+    const disabled = !(title && text);
+    const error = model.logs.hasErrorOccured();
+
+    return h('div#create-log', [
+        h('', [
+            h('h3.black.line-break: auto', 'Title of the log'),
+            h('input#title', {
+                placeholder: 'Enter the title of the log entry...',
+                maxlength: 140,
```

Comment:
You should also include the minLength property.

Reply:
The minlength property has ben added to the input, this has also been incorporated with the button visibility check